data/governments.txt

```diff
@@ -312,6 +312,7 @@ government "Republic"
 	"player reputation" 2
 	"attitude toward"
 		"Alpha" -.3
+		"Korath" -.3
```

Comment:
any reason this is in, by the way?  if anything, this makes getting a positive rep with the Republic as easy as killing 1 Raider
I'd suggest just removing this change entirely, since it's not really necessary

Reply:
I actually made this PR to make it possible to increase your reputation by killing Korath exiles while preventing unintended side effects (like messing up the storyline). But I realized that this is against contributing guidelines (mixing two unrelated commits in one PR), so I am okay if it is needed to separate the two commits to two PRs. Yes, the Republic Navy barely ever meets the Korath exiles, but shouldn't the Republic have high respect for the player for killing some advanced alien pirates raiding human space?